regionmask/defined_regions/climate_indices.py

```diff
@@ -0,0 +1,51 @@
+outlines = dict()
+outlines[1] = ((-120, -5), (-170, -5), (-170, 5), (-120, 5))
+outlines[2] = ((-150, -5), (-90, -5), (-90, 5), (-150, 5))
+outlines[3] = ((210, -5), (160, -5), (160, 5), (210, 5))
+outlines[4] = ((-90, -10), (-80, -10), (-80, 0), (-90, 0))
+outlines[5] = ((180, 20), (-180, 20), (-180, 90), (180, 90))
+outlines[6] = ((180, 20), (-180, 20), (-180, 90), (180, 90))
+outlines[7] = ((180, -20), (-180, -20), (-180, -90), (180, -90))
+outlines[8] = ((180, 20), (-180, 20), (-180, 90), (180, 90))
+outlines[9] = ((180, -15), (-180, -15), (-180, 15), (180, 15))
+
+abbrevs = dict()
+abbrevs[1] = "N34"
+abbrevs[2] = "N03"
+abbrevs[3] = "N04"
+abbrevs[4] = "N12"
```

Comment:
I think I would abbreviate those as NINO34, NINO3, NINO4 and NINO12. Even if that's not really shorter than the name but I think it's more common. If your abbrevs are commonly used  I am of course fine to leave as is.